src/native/libs/System.HybridGlobalization.Native/pal_calendarData.h

```diff
@@ -0,0 +1,120 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include <stdlib.h>
+
+#include "pal_locale.h"
+#include "pal_compiler.h"
+#include "pal_errors.h"
+
+/*
+* These values should be kept in sync with System.Globalization.CalendarId
+*/
+enum
+{
+    UNINITIALIZED_VALUE = 0,
+    GREGORIAN = 1,               // Gregorian (localized) calendar
```

Comment:
There is a lot of duplication between System.Globalization.Native and System.HybridGlobalization.Native.